### PR TITLE
[EXPLORER] Fix zombie window created by CTRL+ALT+DEL

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Purpose

JIRA issue: [CORE-18258](https://jira.reactos.org/browse/CORE-18258)

## Proposed changes
- Fix zombie window created by CTRL+ALT+DEL (Based on patch by [I_Kill_Bugs](https://jira.reactos.org/secure/ViewProfile.jspa?name=I_Kill_Bugs))
- Change `m_DesktopWnd` variable name to `m_ProgMan`

### CODE REVIEW

**Comment by KRosUser2** [comment]
Follow up of https://github.com/reactos/reactos/pull/4651 which was previously approved by closed before merge

File: base/shell/explorer/traywnd.cpp [hunk 1]
```diff
@@ -522,7 +522,7 @@ class CTrayWindow :
 
     HFONT m_Font;
 
-    HWND m_DesktopWnd;
+    HWND m_ProgMgr;
```
Comment (HBelusca) [inline base/shell/explorer/traywnd.cpp:525]: Please use the name `m_ProgMan` since this is the standard name under which it is known.

Comment (TAN-Gaming) [reply base/shell/explorer/traywnd.cpp:525]: OK.

**Review by HBelusca — APPROVED** [review]

**Comment by JoachimHenze** [comment]
I don't want to oppose this PR, but still the question: Does our explorer suffer from this zombie window also when executed on 2k3sp2? And if the answer would be "no", then my question stands "why not?".

**Comment by KRosUser2** [comment]
No zombie in 2K3 : 

[Win2K3-screen0.webm](https://user-images.githubusercontent.com/114949464/204136914-54d1262b-cd1c-4ba5-8b0e-9f8b6eb4725b.webm)

**Comment by binarymaster** [comment]
> No zombie in 2K3

Are you testing with ReactOS Explorer? It seems not.

**Comment by KRosUser2** [comment]
> > No zombie in 2K3
> 
> Are you testing with ReactOS Explorer? It seems not.

Sorry, I missed "our" in the previous comment.

**Comment by HBelusca** [comment]
Any news on the testing status of *our* ReactOS Explorer for this bug on Windows 2003? (both before and after the PR patch?)

**Comment by TAN-Gaming** [comment]
No zombie window on 2k3.
[Win2003-screen0.webm](https://user-images.githubusercontent.com/86486978/206333128-65a08bab-8a27-48b9-a408-0941941f7c51.webm)

**Comment by TAN-Gaming** [comment]
Incorrect fix, closed.

**Pull request closed without merging**